src/main/java/com/epam/test_generator/entities/Case.java

```diff
@@ -151,14 +156,15 @@ public void setStatus(Status status) {
     @Override
     public String toString() {
         return "Case{" +
-            "id=" + id +
-            ", description='" + description + '\'' +
-            ", steps=" + steps +
-            ", creationDate='" + creationDate + '\'' +
-            ", priority=" + priority +
-            ", tags=" + tags +
-            ", status" + status +
-            '}';
+                "id=" + id +
```

Comment:
Instead of concatenation I'd recommend using String.format

Reply:
Done